Sources/Commands/SymbolGraphExtract.swift

```diff
@@ -59,7 +59,7 @@ public struct SymbolGraphExtract {
         let process = Process(
             arguments: arguments,
             outputRedirection: .none,
-            verbose: verbosity != .concise
+            verbose: TSCUtility.verbosity != .concise
```

Comment:
should this take verbosity instead of boolean?

Reply:
`Process` is from `swift-tools-support-core` package though. Can't change it now.

Reply:
okay